### PR TITLE
[DOC] - Update contributing doc

### PR DESCRIPTION
**Pull request by TomDonoghue**
@ryanhammonds - I updated and standardized our Contributing file, here, and across {fooof, lisc, bycycle}. Can you have a a quick read through one of them, and let me know if anything seems weird? They're all ~ the same, so no need to full review each one. Thanks!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,79 +1,131 @@
...
+5. If you want to work on the contribution, follow the [contribution guidelines](#making-a-contribution) to do so
+
+If you are interested in getting involved and helping with the project, a great place to start is to visit the
+[issues](https://github.com/neurodsp-tools/neurodsp/issues) or
+[development](https://github.com/neurodsp-tools/Development) page
+and see if there is anything you would be interested in helping with. If so, join the conversation, and project developers can help get you started.
+
+## Project Scope
+
+All contributions to the module must be within it's scope.
```
Comment (ryanhammonds) [inline CONTRIBUTING.md:56]: its

**Review by ryanhammonds — APPROVED** [review]
This looks great! I found a small typo (sorry for the nitpick) but this looks ready to merge.

File: CONTRIBUTING.md (hunk 1 above)
Comment (TomDonoghue) [reply CONTRIBUTING.md:56]: ahhhhh, apostrophes! Good catch on the typo, thanks!

**Comment by TomDonoghue** [comment]
Thanks for the check @ryanhammonds! I'll fix the typo and do the rounds merging this in

**Pull request merged**